NWL/NWL.py

```diff
@@ -384,17 +384,47 @@ def plot(NWL_mat, phi_pts, theta_pts, num_levels):
 
     levels = np.linspace(np.min(NWL_mat), np.max(NWL_mat), num = num_levels)
     fig, ax = plt.subplots()
-    CF = ax.contourf(phi_pts, theta_pts, NWL_mat, levels = levels)
+    CF = ax.contourf(phi_pts, theta_pts, NWL_mat.T, levels = levels)
     cbar = plt.colorbar(CF)
     cbar.ax.set_ylabel('NWL (MW)')
     plt.xlabel('Toroidal Angle (degrees)')
     plt.ylabel('Poloidal Angle (degrees)')
     fig.savefig('NWL.png')
 
+def area_from_corners(corners):
+    """
+    Calculate an approximation of the area defined by 4 xyz points
+
+    Arguments:
+        corners (4x3 numpy array): list of 4 (x,y,z) points. Connecting the 
+            points in the order given should result in a polygon
+
+    Returns:
+        area (float): approximation of area
+    """
+    # triangle 1
+    v1 = corners[3] - corners[0]
+    v2 = corners[2] - corners[0]
+
+    v3 = np.cross(v1,v2)
+
+    area1 = np.sqrt(np.sum(np.square(v3)))/2
+
+    # triangle 2
+    v1 = corners[1] - corners[0]
+    v2 = corners[3] - corners[0]
+
+    v3 = np.cross(v1,v2)
```

Comment:
This combination of corners is suspicious to me:
```
1 ---- 2
|      |
|      |
0 ---- 3

Current triangle 1:
      2
    /  |
  /    |
0 ---- 3

Current triangle 2:
 1
|   \ 
|     \
0 ---- 3

Reply:
yup, that's my bad. Changing to have the two triangles as follows
0-----1 
|     /
3
and
       1
    /   |
 3----2

Reply:
scratch that, they are actually like this
 0
|   \ 
|     \
3 ---- 2 
and
0----1
 \     |
   \   |
      2